include/mbedtls/config.h

```diff
@@ -2012,6 +2012,20 @@
  */
 //#define MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION
 
+/**
+ * \def MBEDTLS_X509_ALLOW_UNSUPPORTED_CRL_CRITICAL_EXTENSION
+ *
+ * If set, the X509 parser will not break-off when parsing an X509 CRL
+ * and encountering an unknown critical extension. Currently, all extensions
+ * are unknown, including issuingDistributionPoint, which must be critical
+ * and is commonly used.
+ *
+ * \warning Depending on your PKI use, enabling this can be a security risk!
+ *
+ * Uncomment to prevent an error.
+ */
+//#define MBEDTLS_X509_ALLOW_UNSUPPORTED_CRL_CRITICAL_EXTENSION
```

Comment:
Note: if we add such an option:

- it would need to be excluded from the `full` target in `scripts/config.py` (in LTS branches, that would be `scripts/config.pl`)
- we would need a component in `tests/scripts/all.sh` the build and runs the test suites with that option enabled
- we'd probably need to add `!MBEDTLS_X509_ALLOW_UNSUPPORTED_CRL_CRITICAL_EXTENSION` on the `depends_on` line of some existing tests in `tests/suites/test_suite_x509parse.data`
- we'd also want at least one new test in the same file that depends on this option being enabled and demonstrates that unsupported critical extensions are indeed ignored in that case.